01-cloudformation/README.md

```diff
@@ -168,6 +170,8 @@ See [DeletionPolicy](https://aws.amazon.com/premiumsupport/knowledge-center/clou
 
 _How is that different from applying Termination Protection?_
 
+Answer: The permissions required to modify the Termination Protection settings are different than setting the deletion policy.
```

Comment:
These methods are different from applying Termination Protection in that they use IAM policies to prohibit or limit deletions.